ghpc-TB-venn-diagram/css/styles.css

```
@@ -0,0 +1,32 @@
+.tooltip h4 {
+    margin: 0;
+    margin-bottom: .25rem;
+}
+
+.tooltip ul {
+    padding: 0;
+    margin: 0;
+}
+
+.tooltip ul li {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.highcharts-point {
+    stroke: none;
+}
+
+.highcharts-text-outline {
+    stroke-width: 0;
+    fill: white;
+    color: white;
+    visibility: visible;
+    text-shadow: 0px 0px 10px #00000040;
```

For consistency we should remove the px on 0 values.
```suggestion
    text-shadow: 0 0 10px #00000040;
```